raiden/api/rest.py

```diff
@@ -877,6 +881,9 @@ def leave(self, registry_address: typing.PaymentNetworkID, token_address: typing
         ]
         return api_response(result=closed_channels)
 
+    def post_secret_light(self, signed_tx: typing.SignedTransaction):
```

Comment:
this can be confussing, since we have the secret request and secret reveal light

i think post_register_secret_light will be better